wp-content/themes/goonj-crm/templates/dropping-center-landing-page.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/*
+Template Name: dropping center Landing Page
+*/
+?>
+
+<div class="text-center ml-30 mr-30">
+	<h2 class="font-sans fz-20 fw-400 mb-6">Goonj Dropping</h2>
+	<p class="font-sans fz-16 fw-400 mt-0 mb-24">Please go through the details below</p>
+	<ol>
+		<li class="font-sans mb-6 fz-16">
+			Before organising the collection drives, kindly go through our guidelines here - <a href="https://rb.gy/9nfbus" target="_blank">https://rb.gy/9nfbus</a>. Understanding all the guidelines beforehand is important to avoid any inconvenience later.
```

Comment:
We don't need this file, can we use the wordpress pages to show the landing pages so that it can be confiragrable from wp admin.